index.html

```diff
@@ -2,12 +2,77 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
 
+    <div class="wrapper">
+       
+        <div class="container">
+            <div  class="hour">      9:31 </div>
+            <div  class="indicators"> 
+                <img src="./images/status_mobile.png">
+            </div>
+
+        </div>
+    
+        <nav class="nav">            
+            <div class="name"> Edahí  </div>      
+            <div class="hamburger" ><img  src="./images/Hamburger.png">  </div>          
+        </nav>
```

Comment:
- Kindly replace the `nav` tag with a `header`. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. :nerd_face: :nerd_face: 
- Kindly wrap the container with your name inside an `a` tag. This is for three reasons: convention, semantics, and user experience(UX). You can check [this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information.
- Please wrap the hamburger button inside a `nav` tag. The `nav` tag is for semantics reasons.